### PR TITLE
Resource helper chart

### PR DESCRIPTION
**Pull request by MadsHT**
**Description of your changes:**

The resource-helper chart is a helper chart that will
facilitate the creation of ArgoCD project, Resource Quotas
and network policies

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/resource-helper/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Resource-helper
```
Comment (klausenbusk) [inline charts/resource-helper/README.md:1]: If this is only to be used by helm-overdrive (or it is the main user), it should be mentioned here.

Comment (MadsHT) [reply charts/resource-helper/README.md:1]: It is currently it's own purpose, I can mention that?
There is nothing special about it, that ties it to helm-overdrive

Comment (klausenbusk) [reply charts/resource-helper/README.md:1]: I think it is worth mentioning, at least for now.

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Pull request merged**